cmd/skaffold/app/cmd/cmd.go

```diff
@@ -72,6 +72,7 @@ func AddRunDevFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&filename, "filename", "f", "skaffold.yaml", "Filename or URL to the pipeline file")
 	cmd.Flags().BoolVar(&opts.Notification, "toot", false, "Emit a terminal beep after the deploy is complete")
 	cmd.Flags().StringArrayVarP(&opts.Profiles, "profile", "p", nil, "Activate profiles by name")
+	cmd.Flags().StringVarP(&opts.Namespace, "namespace", "n", "", "Run Helm deployments in the specified namespace")
```

Comment:
Would it make sense to make this part of the deploy interface, and make all deployers support it rather than just helm?

Reply:
Thanks for reviewing! It sounds like a good idea, however the kubectl does not support namespace today, so I think it should be first added there, which I think should go in a separate PR.
The current approach does not prevent us from extending this flag to all deployer later.